projects/laji/src/app/shared-modules/document-viewer/document-annotation/document-annotation.component.ts

```diff
@@ -435,8 +430,9 @@ export class DocumentAnnotationComponent implements AfterViewInit, OnChanges, On
 
   private move() {
     this.document = this.result[this.indexPagination].document;
```

Comment:
This line is little strange since I guess that `this.result` can be a list of documents or images now?